### DISCUSSION TITLE
Think about switching to eturnal turn server

### DISCUSSION THREAD

**Discussion by szaimen** [nextcloud/all-in-one#2836] [Ideas] [answered]
See https://github.com/nextcloud/spreed/pull/9828

**Comment by Zoey2936** [comment]
their docker image is still based on alpine v3.16.1, which has many CVEs: https://hub.docker.com/layers/library/alpine/3.16.1/images/sha256-9b2a28eb47540823042a2ba401386845089bb7b62a9637d55816132c4c3c36eb?context=explore
but beside that, are there any advantages?

**Reply by szaimen** [reply]
I had a quick look at it and it looked to be much easier to configure -> so few things that can go wrong that we need to maintain (this is actually the main reason why I suggest this in the first place). However we would need an alpine 3.18 image then of course

**Reply by szaimen** [reply]
Their github repo is this one btw: https://github.com/processone/eturnal. It was not so easy to find XD

**Reply by szaimen** [reply]
Lets see how fast they respond to https://github.com/processone/eturnal/issues/42. If they take too long, I guess we should not switch as we always risk having to wait for this dependency in order to upgrade alpine.

**Reply by Zoey2936** [reply]
> Lets see how fast they respond to [processone/eturnal#42](https://github.com/processone/eturnal/issues/42). If they take too long, I guess we should not switch as we always risk having to wait for this dependency in order to upgrade alpine.

maybe we could recommend them dependencie update tools like renovate/Dependabot

**Reply by szaimen** [reply]
> maybe we could recommend them dependencie update tools like renovate/Dependabot

Yes, do you want to suggest that in the issue?

**Reply by Zoey2936** [reply]
done

**Reply by szaimen** [reply]
thx!

**Reply by Zoey2936** [reply]
```
coturn/coturn                          latest    38d4ced76c6c   About a minute ago   222MB
ghcr.io/processone/eturnal             edge      c1976ceffc38   About an hour ago    35.1MB
```
their docker image is also much smaller

**Reply by szaimen** [reply]
Nice!

**Comment by szaimen** [comment, marked as answer]
Wip in https://github.com/nextcloud/all-in-one/pull/2855
